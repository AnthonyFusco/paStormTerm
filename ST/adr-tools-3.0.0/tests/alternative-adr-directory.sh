adr init alternative-dir
adr new Example ADR

ls .
ls alternative-dir

adr list
