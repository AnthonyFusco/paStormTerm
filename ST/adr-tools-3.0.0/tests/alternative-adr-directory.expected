adr init alternative-dir
alternative-dir/0001-record-architecture-decisions.md
adr new Example ADR
alternative-dir/0002-example-adr.md

ls .
alternative-dir
ls alternative-dir
0001-record-architecture-decisions.md
0002-example-adr.md

adr list
alternative-dir/0001-record-architecture-decisions.md
alternative-dir/0002-example-adr.md
